src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetHelper.cs

```diff
@@ -98,13 +104,15 @@ private static void InitNugetConfig()
         }
     }
 
-    public void ExecuteCmd(string args, string? workingDirectory = null, Action<Process>? additionalProcessConfigCallback = null, int? expectedExitCode = 0, int millisecondTimeout = -1)
+    public void ExecuteCmd(string args, string? workingDirectory = null, Action<Process>? processConfigCallback = null,
+        Action<Process>? processStartCallback = null, int? expectedExitCode = 0, int millisecondTimeout = -1)
     {
         (Process Process, string StdOut, string StdErr) executeResult = ExecuteHelper.ExecuteProcess(
             DotNetPath,
             args,
             OutputHelper,
-            configure: (process) => configureProcess(process, workingDirectory),
+            configureCallback: (process) => configureProcess(process, workingDirectory),
+            startCallback: processStartCallback,
```

Comment:
Is `processStartCallback` not used yet and meant for future extension?

Reply:
Ah, good catch. I added that during development but then found out I didn't need it. I'll get rid of it.